src/contents/en-gb/parsec.yaml

```diff
@@ -0,0 +1,66 @@
+pageIntro:
+  title: What is PARSEC?
+  desc: PARSEC is a Protocol for Asynchronous, Reliable, Secure and Efficient Consensus and is the world's first open source ABFT protocol that can be used in a truly permissionless setting created by MaidSafe.

```

Comment:
font here doesn't match the rest of the page

Reply:
it's the page header - I'll address (btw - just notes for me as I'm going through these)

Reply:
format changed to match